docs/modules/policies/pages/resource_policies.adoc

```diff
@@ -39,3 +39,34 @@ resourcePolicy:
 <6> Rules can also refer directly to static roles.
 <7> Optional name for the rule.
 
+== Another example
+[source,yaml,linenums]
+----
+---
+apiVersion: api.cerbos.dev/v1
+resourcePolicy:
+  importDerivedRoles:
+  - my_derived_roles
+  resource: leave_request
+  globals: <1>
+    pending_approval: ("PENDING_APPROVAL")
+    principal_location: |-
+      (P.attr.ip_address.inIPAddrRange("10.20.0.0/16") ? "GB" : "")
+  rules:
+  - actions: ["approve"]
+    condition:
+      match:
+        expr: request.resource.attr.status == globals.pending_approval
+    derivedRoles:
+      - direct_manager
+    effect: EFFECT_ALLOW
+  - actions: ["delete"]
+    condition:
+      match:
+        expr: request.resource.attr.geography == globals.principal_location
+    derivedRoles:
+    - direct_manager
+    effect: EFFECT_ALLOW
+----
+<1> This resource policy includes global variables section. Each variable is evaluated before evaluating any rule condition.
+A variable expression can contain anything that condition expression can have.
```

Comment:
I would just add a `globals` section to the policy above and annotate it. The idea is to show the anatomy of a policy with all the possible fields you can have on one. Having two examples adds noise and breaks that flow.

Then I would add a new section to the conditions documentation and explain how to use `globals` when writing conditions. This example can be used there to make it clearer. 

It just occurred to me that `globals` should be supported in both derived roles and principal policies as well. Let's not forget those in this PR.

Reply:
Done